viz.yaml

```diff
@@ -109,28 +109,28 @@ process:
     location: cache/process/lastUpdated.yaml
     scripts: scripts/process/lastUpdated.R
     depends:
-      data: aggregate_ga
+      data: fetchGA
     mimetype: text/yaml
     reader: md # need a better plain text reader
   -
     id: sessions_and_new_users
     processor: sessions_and_new_users 
     scripts: scripts/process/sessions_and_new_users.R
-    depends: aggregate_ga
+    depends: fetchGA
```

Comment:
poop, I can change those too, or do you want to try to revive the memory problem by just using `distinct` in the regular code?